### ISSUE TITLE
Lazer font improvements

### ISSUE THREAD

**Issue by ToIGLAK** [ppy/osu#2779]
I know this is a ridiculous bug (actually who write Lenny face in hobbies :D). So Lazer Font doesn't have that weird characters used in (͡° ͜ʖ ͡°). I see two solutions for that. First: adding that character to Lazer Font; Second: using similar font that has those and much more other strange characters, in case when Lazer detect their lack.

Also, I found stuff like: "last seen" and "joined" mix oddly with the polish equivalent of that word.
For example: "Last seen **przed 7 minutami**" where correct form is "Last seen **7 minutes ago**".

Correct: ![should be](https://user-images.githubusercontent.com/37531387/41256365-a6452e4c-6dc9-11e8-9021-90689c50b9af.png)

Wrong: ![bug](https://user-images.githubusercontent.com/37531387/41256372-aa244f48-6dc9-11e8-9b3e-8121907c85b2.png)

Hope I help :)

**Comment by Aergwyn** [comment]
This seems to be a web issue. Maybe you want to open one [over there](https://github.com/ppy/osu-web/issues).

**Comment by ToIGLAK** [comment]
Lenny Face looks correctly on web. Lazer do something wrong. 

Web: ![correct](https://user-images.githubusercontent.com/37531387/41256670-98bd6cc0-6dca-11e8-8921-c962b5ad933a.png)

Lazer: ![bug1](https://user-images.githubusercontent.com/37531387/41256676-9e222430-6dca-11e8-9500-027e2879b0c1.png)

**Comment by Aergwyn** [comment]
Oh I see. I misunderstood. Alright.

**Comment by peppy** [comment]
Yes, we currently only have unicode ranges added for languages, not silly characters. We'll need to do this at some point.
